docs/source-1.0/spec/aws/aws-json-1_1-protocol.rst

```diff
@@ -137,4 +137,28 @@ The following example defines a service that requires the use of
 .. |protocol test link| replace:: https://github.com/awslabs/smithy/tree/main/smithy-aws-protocol-tests/model/awsJson1_1
 .. include:: aws-json.rst.template
 
+
+--------------------
+Error shape renaming
+--------------------
+
+By default, Smithy permits renaming shapes to disambiguate shape ID conflicts in
+the :ref:`service closure <service-closure>` via the ``rename`` property.
+
+However, services using ``awsJson1_1`` protocol are not
+allowed to rename error shapes (shapes with :ref:`error trait <error-trait>` applied).
+
+Client-side implementation relies on the response body field ``code`` or ``__type`` to resolve the error type.
+Server-side implementation of ``awsJson1_1`` protocol will only send the ``shape name`` for the response body field.
```

Comment:
Did you verify this?

Reply:
I have confirmed with @adamthom-amzn on this.